demos/security_barrier_camera_demo/main.cpp

```diff
@@ -336,8 +336,11 @@ void Drawer::process() {
         out << "Mean overall time per all inputs: " << std::fixed << std::setprecision(2) << std::setw(6);
         const auto t1 = std::chrono::steady_clock::now();
         uint64_t frameCounter = context.frameCounter;
-        const ms meanOverallTimePerAllInputs = std::chrono::duration_cast<ms>((t1 - context.t0)
-                                               * context.readersContext.inputChannels.size()) / frameCounter;
+        ms meanOverallTimePerAllInputs = std::chrono::duration_cast<ms>((t1 - context.t0)
+                                               * context.readersContext.inputChannels.size());
+        if (0 != frameCounter) {
```

Comment:
This seems strange. Is it actually possible for this code to be executed when `frameCounter` is 0?

Reply:
starting and immediately stopping can result in zero frame processing. For example if VideoCapture will claim there are no frames in a file.

Reply:
Well, I am not sure I can reproduce this case, because providing empty file results in "Cannot open" error, but I am still not sure that it is not impossible.

Reply:
Well, OK. But isn't it misleading to print the time per frame (as well as FPS) when there haven't been any frames? Maybe a better fix would be to place the whole block from the declaration of `meanOverallTimePerAllInputs` to the FPS output into the `if`.

Reply:
You are right. Fixed